proof-market/application-developers/tutorials/managing-statements.mdx

```diff
@@ -0,0 +1,61 @@
+import ApiSchema from '@theme/ApiSchema';
+
+# Managing statements
+
+This tutorial explains what statements are and their role in application development.
+
+## Definition
+
+Statements are circuits with additional metadata such as a name and an ID.
+
+The primary purpose of statements is to 'wrap' circuits inside a tradeable entity that can be used to match proof requesters and proof producers. 
+
+## Attaining available statements
+
+If reusing an existing circuit is sufficient for the app, check the list of available statements to see whether there exist ready-made statements.
+
+To do so, send a [**`GET` request to `/statement`**](../../api-reference#operation/GetByFilter), which should return a list of `StatementItem` objects.
+
+<ApiSchema pointer="#/components/schemas/StatementItem"/>
+
+If there is no need to create a new statement, skip the next step.
+
+## Creating a new statement (Optional)
+
+To create a new statement, perform a [**`POST` request to `/statement`**](../../api-reference#operation/CreateStatement) with the following request body.
+
+<ApiSchema pointer="#/components/schemas/CreateStatementRequest"/>
+
+The response will contain a newly generated `StatementItem` object.
+
+:::tip[Circuit inputs]
+
+When creating a new statement, specify circuit inputs in the `"input_description"` key as a sequence of values and types separated by spaces. Consider these examples.
+
+- `2 int 5 int`
+- `10 field 98 int`
+
+:::
+
+## Accessing statement statistics
+
+After selecting an existing statement or creating a new one, the app must attain statistics about the requests made by other users and the proposals made by proof producers.
+
+Perform a [**`GET` request to `/book/{statementId}`**](../../api-reference#operation/StatementBook) to produce the below response.
+
+<ApiSchema pointer="#/components/schemas/StatementBook"/>
+
+## Attaining user balance (Optional)
```

Comment:
I feel that we need to have a separate tutorial how to stake to the contract expalining how to get contract details and network and how to stake and why to stake